src/permissions/PermissionController.test.ts

```diff
@@ -4384,6 +4384,76 @@ describe('PermissionController', () => {
       expect(hasPermissionsSpy).toHaveBeenNthCalledWith(2, 'foo');
     });
 
+    it('action: PermissionController:hasAllPermissions', () => {
```

Comment:
I was expecting to see both the public method and the action tested. The coverage threshold doesn't demand it, but that's just a guideline. The public method is part of the public API of this package, so we want to ensure its behavior is fully tested so we don't break it accidentally and unknowingly ship a breaking change.

Reply:
That would be a good opportunity to break this into smaller tests as well. Right now it's not clear what this test is trying to prove, there is too much going on. The case of calling `hasAllPermissions` when the subject has none, some, or all of the target permissions could be three separate test cases. Then the test description can make clear exactly what's being tested.

Probably worth adding a test case for calling `hasAllPermissions` with an empty array as well. And for when the subject has additional permissions.